api/src/EthBridge.ts

```diff
@@ -0,0 +1,93 @@
+import { SubmittableExtrinsic } from '@polkadot/api/types';
+import { GearApi } from './GearApi';
+import { ISubmittableResult } from '@polkadot/types/types';
+import { HexString, Proof } from './types';
+import { AuthoritySetHashError, ClearTimerError, GetQueueMerkleRootError } from './errors';
+
+export class GearEthBridge {
+  constructor(private api: GearApi) {}
+
+  async authoritySetHash(): Promise<HexString> {
+    const result = await this.api.query.gearEthBridge.authoritySetHash();
+
+    try {
+      const hash = result.unwrap();
+      return hash.toHex();
+    } catch (err) {
+      throw new AuthoritySetHashError(err);
+    }
```

Comment:
There should be methods `.is_some` and `.is_none` if I'm not mistaken. Then it can look like this
```javascript
if (result.is_none()) {
  throw new AuthroitySetHashNotFound(); // cuz if it's `none` it literally not found
}
return result.unwrap().toHex()
```